src/sockets/tests/matchmaking.test.ts

```diff
@@ -0,0 +1,64 @@
+import { Matchmaking } from '../matchmaking';
+
+describe('Matchmaking', () => {
+    let matchmaking: Matchmaking;
+
+    beforeEach(() => {
+        matchmaking = new Matchmaking();
+    });
+
+    it('should allow a user to be added to the queue', () => {
+        matchmaking.addUserToQueue('pronub');
+
+        expect(matchmaking.userQueue).toContain('pronub');
+    });
+
+    it('should remove a user from the queue', () => {
+        matchmaking.addUserToQueue('pronub');
+
+        expect(matchmaking.userQueue).toContain('pronub');
+
+        matchmaking.removeUser('pronub');
+
+        expect(matchmaking.userQueue).not.toContain('pronub');
```

Comment:
Replace test with:

* Add 5 users
* No match
* Add 6th user
* Match
* Try to find another match, should be empty